test/ref-list.json

```diff
@@ -44504,7 +44504,6 @@
     "MEDIAQUERIES-20170905",
     "MEDIAQUERIES-20200721",
     "MEDIAQUERIES-20211225",
-    "MEDIAQUERIES-3",
     "MEDIAQUERIES-3-20010404",
```

Comment:
So we can’t remove this test without breaking backwards compat. This needs to continue pointing to MQ3.

Reply:
If I put it back and not the alias, then I get the error:

>  1) Previous references are maintained
>       Reference [[MEDIAQUERIES-3]] is maintained.:
>     AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:
>
>  assert.ok(current.get(key)[key])
>
>      at Context.<anonymous> (test\no-ref-loss.js:12:20)
>      at processImmediate (internal/timers.js:461:21)

Is that okay to ignore?

Reply:
Ah, okay. I'll leave it to the experts, then!

Reply:
It isn’t expertise, it’s tinkering and patience.

Reply:
Sure, my tinkering has just hit a wall in that I don't understand what backwards compatibility is breaking, so I don't even know what else to tinker with. I only know enough to follow what a test run returns or I'd offer to help more.

Reply:
Apologies for the lack of clarity. Specref's contract has always been that that although references might get updated, they're never removed. MEDIAQUERIES-3 should continue to point to something. The REC maybe?

Reply:
> MEDIAQUERIES-3 should continue to point to something. The REC maybe?

Won't this be restored once the new w3c data gets pulled into the w3c.json file? The new REC has the shortname mediaqueries-3, so I'm assuming it will result in an entry like:

```
"mediaqueries-3": {
   ...
   "versions": [
      "20200505": {
         ...
      }
   ]
}
``` 

which would only make for a temporary gap when the reference won't exist, but I'm not sure how else to work around that.